frontend/src/fixtures/ucsbSubjectsFixtures.js

```diff
@@ -0,0 +1,44 @@
+
+const ucsbSubjectsFixtures = {
+    oneSubject: {
+        "id" : "randomId",
+        "subjectCode": "randomSubjectCode",
+        "subjectTranslation" : "randomTranslation",
+        "deptCode" : "randomDeptCode",
+        "collegeCode" : "randomCollegeCode",
+        "relatedDeptCode" :"randomRelatedDeptCode",
+        "inative" : false
+    },
+    threeSubjects: [
+        {
+            "id" : "1",
+            "subjectCode": "0001",
+            "subjectTranslation" : "randomTranslation1",
+            "deptCode" : "randomDeptCode1",
+            "collegeCode" : "randomCollegeCode1",
+            "relatedDeptCode" :"randomRelatedDeptCode1",
+            "inative" : false
+        },
+        {
+            "id" : "2",
+            "subjectCode": "1003",
+            "subjectTranslation" : "randomTranslation2",
+            "deptCode" : "randomDeptCode2",
+            "collegeCode" : "randomCollegeCode2",
+            "relatedDeptCode" :"randomRelatedDeptCode2",
+            "inative" : false
+        },
+        {
+            "id" : "3",
+            "subjectCode": "2016",
+            "subjectTranslation" : "randomTranslation3",
+            "deptCode" : "randomDeptCode3",
+            "collegeCode" : "randomCollegeCode3",
+            "relatedDeptCode" :"randomRelatedDeptCode3",
+            "inative" : false
+        }
+    ]
+};
```

Comment:
I wonder if we could use real values from the API for these fixtures instead of complete fake ones?

Reply:
Will add real values instead of the random ones.

Reply:
Looks good